301-quarkus-vertx-kafka/src/test/java/io/quarkus/qe/vertx/kafka/StrimziKafkaResource.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.qe.vertx.kafka;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.Network;
+import org.testcontainers.lifecycle.Startables;
+import io.strimzi.StrimziKafkaContainer;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class StrimziKafkaResource implements QuarkusTestResourceLifecycleManager {
+
+    private static final Logger LOG = Logger.getLogger(ConfluentKafkaResource.class);
+
+    private StrimziKafkaContainer kafkaContainer;
+    private SchemaRegistryContainer schemaRegistry;
+
+    @Override
+    public Map<String, String> start() {
+        Network network = Network.newNetwork();
+
+        kafkaContainer = new StrimziKafkaContainer("0.18.0-kafka-2.5.0").withNetwork(network);
```

Comment:
Just a note, not subject of this PR.

: We need to figure out the right way how to track image names and image versions., if we need to customize them, how often should be updated, how to remind ourselves about updating them, etc.